include/quda_cuda_api.h

```diff
@@ -5,6 +5,8 @@
 #include <cuda_runtime.h>
 #include <quda_cuda_api.h>
 
+#define qudaStream_t cudaStream_t
```

Comment:
does this need to be a define. Can we get away with a `using` ?

Reply:
This was for expediency for this PR only.  I'm not sure if `using` works, but I can try.

Reply:
@maddyscientist It compiles with `using`, I'll do a run time check in a minute.

Reply:
And it runs.